Code/GraphMol/MolOps.h

```diff
@@ -212,25 +234,32 @@ RDKIT_GRAPHMOL_EXPORT unsigned int getMolFragsWithQuery(
        - the caller is responsible for <tt>delete</tt>ing the pointer this
    returns.
  */
-RDKIT_GRAPHMOL_EXPORT ROMol *addHs(const ROMol &mol, bool explicitOnly = false,
-                                   bool addCoords = false,
-                                   const UINT_VECT *onlyOnAtoms = nullptr,
-                                   bool addResidueInfo = false);
+inline ROMol *addHs(const ROMol &mol, bool explicitOnly = false,
+                    bool addCoords = false,
+                    const UINT_VECT *onlyOnAtoms = nullptr,
+                    bool addResidueInfo = false) {
+  AddHsParameters ps{explicitOnly, addCoords, addResidueInfo};
+  RWMol *res = new RWMol(mol);
+  addHs(*res, ps, onlyOnAtoms);
+  return static_cast<ROMol *>(res);
+}
 //! \overload
 /// modifies the molecule in place
-RDKIT_GRAPHMOL_EXPORT void addHs(RWMol &mol, bool explicitOnly = false,
-                                 bool addCoords = false,
-                                 const UINT_VECT *onlyOnAtoms = nullptr,
-                                 bool addResidueInfo = false);
+inline void addHs(RWMol &mol, bool explicitOnly = false, bool addCoords = false,
+                  const UINT_VECT *onlyOnAtoms = nullptr,
+                  bool addResidueInfo = false) {
+  AddHsParameters ps{explicitOnly, addCoords, addResidueInfo};
+  addHs(mol, ps, onlyOnAtoms);
+}
 
 //! Sets Cartesian coordinates for a terminal atom.
 //! Useful for growing an atom off a molecule with sensible
 //! coordinates based on the geometry of the neighbor.
 /*!
-    NOTE: this sets appropriate coordinates in all of the molecule's conformers.
-    \param mol       the molecule the atoms belong to
-    \param idx       index of the terminal atom whose coordinates are set
-    \param otherIdx  index of the bonded neighbor atom
+    NOTE: this sets appropriate coordinates in all of the molecule's
+   conformers. \param mol       the molecule the atoms belong to \param idx
+   index of the terminal atom whose coordinates are set \param otherIdx  index
+   of the bonded neighbor atom
```

Comment:
Am I seeing github diff artifact, or were those `\param` pulled into the previous line ?

If it's real, we should probably put the `\param`s back into new lines.